File: src/targets/ocaml/cohttp/fixtures/cookies.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

let uri = Uri.of_string "https://httpbin.org/cookies" in
let headers = Header.add (Header.init ()) "cookie" "foo=bar; bar=baz" in

Client.call ~headers `GET uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)

File: src/targets/node/fetch/fixtures/cookies.js
const url = 'https://httpbin.org/cookies';
const options = {method: 'GET', headers: {cookie: 'foo=bar; bar=baz'}};

fetch(url, options)
  .then(res => res.json())
  .then(json => console.log(json))
  .catch(err => console.error(err));